plugins/flytekit-vscode/README.md

```diff
@@ -39,3 +39,59 @@ If users want to skip the vscode downloading process at runtime, they have the o
 + RUN tar -xzf /tmp/code-server/code-server-4.18.0-linux-amd64.tar.gz -C /tmp/code-server/
 + ENV PATH="/tmp/code-server/code-server-4.18.0-linux-amd64/bin:${PATH}"
 ```
+
+## More Examples
+
+```python
+from flytekit import task, workflow
```

Comment:
could we also add this example to flytesnacks?

Reply:
sure i have added that to our progress tracker: https://docs.google.com/document/d/1NkjCbYoOqz-b9l-3UxZnUzpefsZiA7SC-mD5_Dp6dMQ/edit#bookmark=id.4fpr1ayjw6gj

Reply:
thank you